crates/bevy_sprite/src/mesh2d/wireframe2d.wgsl

```diff
@@ -0,0 +1,12 @@
+#import bevy_sprite::mesh2d_vertex_output::VertexOutput
+
+struct WireframeMaterial {
+    color: vec4<f32>,
+};
+
+@group(2) @binding(0)
+var<uniform> material: WireframeMaterial;
```

Comment:
```suggestion
@group(2) @binding(0) var<uniform> material: WireframeMaterial;
```

Tiny nitpick, the 3d wireframe shader doesn't do that apparently but we now use this convention for all internal shaders.